api/problem/create.py

```
@@ -0,0 +1,40 @@
+#   FUNCTION
+#
+
+# FUNCTION -> DECLARATION
+def output(request: HttpRequest) -> HttpResponse:
+    # DECLARATION -> EXTENSIONS
+    from website.extensions import Response; Response(request)
+    from website.extensions import database; database.init() 
+    from website.extensions import post; post.init() 
+    from website.extensions import time; time.init()
+    # DECLARATION -> ARGUMENT CHECKS
+    if not post.checks("Kid", "Pmeta", "Psolution", "Pdataen", "Pdataes", "Pdatade"): return SUG.REQ.RES.error(1)
```

Ruff (0.11.9)

Multiple statements on one line (colon)

(E701)